common/filetoolkitwithundo.cpp

```diff
@@ -111,16 +111,20 @@ bool FileToolkitWithUndo::bindMount(const std::string &src,
                                     bool readOnly,
                                     bool enableWriteBuffer)
 {
-    unsigned long flags = MS_BIND;
+    unsigned long flags =  0; //MS_BIND;
     std::string fstype;
-    const void *data = nullptr;
     int mountRes;
 
     if (!existsInFileSystem(src)) {
         log_error() << src << " does not exist on the host, can not bindMount";
         return false;
     }
 
+    if (!existsInFileSystem(dst)) {
+        log_error() << src << " does not exist on the host, can not bindMount";
+        return false;
+    }
```

Comment:
Actually no... code copy, but checking for src in the first test, then dst, but forgot to replace src with dst in the log message. I've fixed that. I hope that is good enough?